src/result.cpp

```diff
@@ -20,6 +21,14 @@ double result_array::max() const {
 
 double result_array::avg() const { return std::accumulate(m_data.begin(), m_data.end(), 0.0) / m_data.size(); }
 
+double result_array::stdev() const { 
+    std::vector<double> c_data = m_data; 
```

Comment:
I think a for loop would be easier to follow here and eliminate the need of copying m_data:
```
double mean = avg();
double sum = 0;
for (double d : m_data)
    sum += (d - mean) * (d - mean);
return std::sqrt(sum / m_data.size());
```

Reply:
I wanted to keep the same style as the previous functions. I agree though, this looks way better. I changed it.